### PR TITLE
Polyfills: Move polyfills out of global namespace

### PR DESCRIPTION
**Pull request by alexisgrow**
Move away from babel/polyfill, upgrade core-js to version 3.

TEST=auto,manual

Pull client sites and test that on IE11 the polyfill works.
Initialize answers on dev test sites using all components.

### CODE REVIEW

File: conf/gulp-tasks/library.gulpfile.js
Comment (alexisgrow) [inline conf/gulp-tasks/library.gulpfile.js:76]: note to self: 
"useBuiltIns and @babel/plugin-transform-runtime are mutually exclusive. Both are used to add polyfills: the first adds them globally, the second one adds them without attaching them to the global scope."

so removing this in favor of the transform-runtime plugin below 

src: https://github.com/babel/babel/issues/10271#issuecomment-528379505

File: conf/gulp-tasks/library.gulpfile.js
Comment (alexisgrow) [inline conf/gulp-tasks/library.gulpfile.js:75]: note to self:

stdout says the `corejs` option is useless/irrelevant w/o the `useBuiltIns` option, removing both

File: package.json
Comment (alexisgrow) [inline package.json]: note to self: this is a separate dependency thing that is breaking the circleci build, it should not be merged as part of the polyfill item

File: conf/gulp-tasks/library.gulpfile.js
Comment (alexisgrow) [reply conf/gulp-tasks/library.gulpfile.js:76]: second source, maintainer saying docs need to be updated https://github.com/babel/babel/issues/10121#issuecomment-505005586

File: package.json
Comment (alexisgrow) [inline package.json]: i actually don't think this is relevant/necessary any more

File: package.json
Comment (tmeyer2115) [reply package.json]: Should this line be removed, then?

Comment (alexisgrow) [reply package.json]: Yeah I was going to ask you about that! It started breaking our CircleCI build, so we fixed w/ this PR (https://github.com/yext/answers/pull/505). The fix is merged into the v13.0 branch, but is still an issue on this v12.2 branch. Should we pull https://github.com/yext/answers/pull/505 into this (v12.2) branch?

**Comment by b1lly** [comment]
LGTM if LGTY @tmeyer2115

File: package.json
Comment (alexisgrow) [reply package.json]: Removing!

**Review by tmeyer2115 — APPROVED** [review]

**Pull request merged**
